include/t1ha0_ia32aes_b.h

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Stupid question, looks like this is the zLIB license, compatible with GPL, but so far we have apache everywhere else.. could that pose a problem for users?

Reply:
Very good question. Will check it out

Reply:
https://www.apache.org/legal/resolved.html states it’s compatible